tests/test_fisher.py

```diff
@@ -71,9 +71,8 @@ def test_fisher():
         operators_to_keep = np.array(["Op1", "Op2", "Op3"])
         dataset = load_datasets(
             commondata_path,
-            datasets=["data_test5"],
+            datasets={"data_test5": {"order": "NLO"}},
```

Comment:
Long-term idea: a simple converter of the runcards from the old format to the new one could be useful.